README.md

```diff
@@ -4,22 +4,22 @@ Trill is a high-performance one-pass in-memory streaming analytics engine from M
 
 # Getting Started
 
+## Building Trill
 1. Of course, the sources are right here!
-2. You can get binaries from our [NuGet feed](https://www.nuget.org/packages/Trill/).
-3. You can also check out samples of Trill usage at our [samples repository](https://github.com/Microsoft/TrillSamples).
+1. Clone the Repo and make sure you have Visual Studio 2017 installed
+1. Open `Trill.sln` solution available in `./Sources` with Visual Studio 2017 
+1. Build Trill
```

Comment:
These are all numbered "1", can you increment them?

Reply:
AFAIK when rendered the list will automatically generate the numbers. Anyway, I've just set the correct numbers manually now :)

Reply:
Ah good to know

---
In reply to: [246571358](https://github.com/Microsoft/Trill/pull/38#discussion_r246571358) [](ancestors = 246571358)